astro/src/content/docs/get-started/start-here/step-1.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Step 1 - Install"
+description: Learn how to use FusionAuth by first installing it on your development machine.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
+---
+import DownloadWidget from 'src/components/download/DownloadWidget.astro';
+
+<div className="border border-green-700 px-3 rounded-md">
+  <i className="fa-duotone fa-stopwatch fa-2x mr-2 text-green-700 align-middle"/> Estimated time to complete this step: **2 minutes**
+</div>
+
+## Install FusionAuth
+
+First, you need to get FusionAuth up and running. The simplest way to run FusionAuth is to install it on your development machine (which is probably the laptop you are reading this doc on). The benefit of running FusionAuth locally is that you don't need internet to use FusionAuth and you can also automate unit, integration, and functional tests against it.
+
+Plus, you'll likely want to run FusionAuth in your staging environment, and probably in your CI/CD pipeline as well. Learning to install FusionAuth now is a great place to start.
+
+Pick your preferred method of installation below. And remember, these are not exhaustive. FusionAuth also provides platform-specific options and services such as DEB and RPM packages. For a list of all our download and installation options, visit https://fusionauth.io/download.
+
+<DownloadWidget kickstartEnabled={true} kickstartName={"get-started"}/>
+
+## Open the web UI
+
+Open a browser to http://localhost:9011 and you'll be presented with the FusionAuth login page. The installation instructions above used [Kickstart](/docs/get-started/download-and-install/development/kickstart), which created an admin user:
+
+* Login id: **admin@example.com**
+* Password: **password**
+
+Go ahead and log into the FusionAuth admin UI now by clicking the lock icon in the top right corner.
+
+**NOTE:** While not required, you might want to log out of the FusionAuth admin before proceeding to the next step. You can do this by clicking the `Logout` button in the top right corner.
+
+## Next steps
+
+<span className="inline-block mr-6 font-bold">Ready for the next step?</span> [Step 2 - Login button \>](step-2)
```

Comment:
At some point it would be nice to drive this stepwise nav more dynamically so we can easily edit/add steps, change step names, etc.

Reply:
Well, that's the tricky part. The more steps we add, the more complex and cumbersome this whole section gets. And eventually it stops being a "Getting Started" guide and becomes a "Do Everything" guide. But point taken on making this a bit dynamic to ensure that the numbering is accurate and the links work.